js/whosehouse.js

```diff
@@ -30,9 +30,11 @@ function WhoseHouse() {
 var wh = new WhoseHouse();
 
 var baselineRotation = 0;
-
+var transition_end_events = 'transitionend webkitTransitionEnd \
```

Comment:
Typical style for Javascript variable names is camelCase (`transitionEndEvents`)

Reply:
ah right...